integration-test/src/main/java/com/sequenceiq/it/cloudbreak/SshUtil.java

```diff
@@ -0,0 +1,99 @@
+package com.sequenceiq.it.cloudbreak;
+
+import java.io.IOException;
+import java.util.concurrent.TimeUnit;
+
+import org.apache.commons.lang3.tuple.Pair;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import net.schmizz.sshj.SSHClient;
+import net.schmizz.sshj.common.IOUtils;
+import net.schmizz.sshj.connection.channel.direct.Session;
+import net.schmizz.sshj.transport.verification.PromiscuousVerifier;
+
+
+public class SshUtil {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(com.sequenceiq.it.cloudbreak.SshUtil.class);
+
+    public Boolean ssh(String host, String defaultPrivateKeyFile, String sshCommand, String checkType, String value) {
+        SSHClient sshClient = null;
+        boolean result = false;
+        try {
+            sshClient = createSSHClient(host, 22, "cloudbreak", defaultPrivateKeyFile);
+            Pair<Integer, String> cmdOut = executeSshCommand(sshClient, sshCommand);
+            LOGGER.info("Ssh command status code and output: " + cmdOut.toString());
+            result = cmdOut.getLeft() == 0 && checkCommandOutput(cmdOut, checkType, value);
+        } catch (Exception ex) {
+            LOGGER.error("Error during remote command execution", ex);
+        } finally {
+            try {
+                if (sshClient != null) {
+                    sshClient.disconnect();
+                }
+            } catch (Exception ex) {
```

Comment:
Would be nicer to handle exceptions on the caller side (once) instead of this try-catch-final-try-catch monster :)

Reply:
This is a single operation. If the caller would handle the exception, we would have to return the ssh connection object to be able to safely close the connection. That way the functionality would be stored in two different classes.